framework/PostProcessors/Metric.py

```diff
@@ -59,6 +61,12 @@ class cls.
     TargetsInput = InputData.parameterInputFactory("Targets", contentType=InputData.StringType)
     TargetsInput.addParam("type", InputData.StringType)
     inputSpecification.addSub(TargetsInput)
+    MultiOutputInput = InputData.parameterInputFactory("multiOutput", contentType=InputData.StringType)
+    inputSpecification.addSub(MultiOutputInput)
+    WeightInput = InputData.parameterInputFactory("weight", contentType=InputData.StringType)
```

Comment:
camelBack 
```MultiOutputInput, WeightInput, PivotParameterInput```

Reply:
you should EnumType here...so you do not need to check below if you got a correct option

Reply:
updated.